tests/source/assignment.rs

```diff
@@ -16,7 +16,7 @@ fn main() {
 
 
     // #2791
-    let x = 2;;;;
```

Comment:
why are the trailing semicolons removed ?

Reply:
Because they are...unnecessary? 😄 

These trailing semicolons are not part of the AST, so removing them won't affect the semantic of the code.

Reply:
But this is the **source** file. It's ok that you remove them from the **target** file.

Reply:
Oops, you are right...thanks for the catch!